engine/modelc/src/modelimporter_gltf.cpp

```diff
@@ -412,17 +427,24 @@ static void LoadPrimitives(Model* model, cgltf_data* gltf_data, cgltf_mesh* gltf
         {
             cgltf_attribute* attribute = &prim->attributes[a];
             cgltf_accessor* accessor = attribute->data;
-            //printf("  attributes: %s   index: %u   type: %s  count: %u\n", attribute->name, attribute->index, GetAttributeTypeStr(attribute->type), (uint32_t)accessor->count);
 
             mesh->m_VertexCount = accessor->count;
 
             uint32_t num_components = (uint32_t)cgltf_num_components(accessor->type);
             uint32_t desired_num_components = num_components;
 
+            //printf("  attributes: %s   index: %u   type: %s  count: %u desired_num_components: %u\n", attribute->name, attribute->index, GetAttributeTypeStr(attribute->type), (uint32_t)accessor->count, desired_num_components);
+
+            float default_value_f = 0.0f;
             if (attribute->type == cgltf_attribute_type_tangent)
             {
                 desired_num_components = 3; // for some reason it give 4 elements
             }
+            else if (attribute->type == cgltf_attribute_type_color)
+            {
+                desired_num_components = 4; // We currently always store vec4
+                default_value_f = 1.0f; // for the alpha channel
```

Comment:
Perhaps in the future we'll support only rgb colors, but for now this is the quickest way forward.